src/main/java/org/broadinstitute/hellbender/tools/copynumber/DetermineGermlineContigPloidy.java

```diff
@@ -31,9 +29,13 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
+import static org.broadinstitute.hellbender.tools.copynumber.arguments.CopyNumberArgumentValidationUtils.streamOfSubsettedAndValidatedReadCounts;
+
 /**
  * Determines the integer ploidy state of all contigs for germline samples given counts data. These should be either
- * HDF5 or TSV count files generated by {@link CollectReadCounts}.
+ * HDF5 or TSV count files generated by {@link CollectReadCounts}; TSV files may gzipped, but must then have filenames
```

Comment:
I think gzipped implies block compression but I'm not actually sure which implementation HTSJDK is compatible with.  Perhaps I'll specify compression with bgzip, since that is what we will presumably use?  Index is only expected when streaming.

Reply:
Yes sorry, I meant block compression in the way that bgzip implements it. Specifying bgzip would be clearer.